### DISCUSSION TITLE
Update YouTube Video

### DISCUSSION THREAD

**Discussion by Meche697** [rejetto/hfs#255] [Ideas]
I haven't used this in multiple years ...
How about a new HOW-TO YouTube video? Most videos are 10 yrs old and the newer ones are not in English. I remember having to download a DDNS to set up ... again, long time.

**Comment by rejetto** [comment]
how to videos are good, hopefully someone will make them.
On the new wiki anyone can contribute, with guides and anything else. https://github.com/rejetto/hfs/wiki
DDNS is not mandatory, and old HFS had integrated support, so no extra download was necessary.
